seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -4184,6 +4184,9 @@
         "Here's a sample object:",
```

Comment:
Basically, I think this needs to be rearranged and tweaked some. Sorry for not doing this earlier.

```
You may have heard the term object before.

Objects are similar to arrays, except that instead of using indexes to access and modify their data, you access the data in objects through properties.

Objects are useful for storing data in a structured way, and can represent real world objects, like a cat.

Here's a sample cat object:
<cat object code>

In this example, all the properties are stored as strings, such as - "name", "legs", and "tails". However, you can also use numbers as properties or omit the quotes for single-word string properties, as seen below. Note! if your object has any non-string properties, JavaScript will automatically typecast them as strings.
<another object code>

<hr>

Make an object called myDog object that contains the properties "name", "legs", "tails" and "friends".

You can set these object properties to whatever values you want, as long "name" is a string, "legs" and "tails" are numbers, and "friends" is an array.
```